test/js/web/websocket/websocket-client-short-read.test.ts

```diff
@@ -0,0 +1,107 @@
+import { TCPSocketListener } from "bun";
+import { describe, test, expect } from "bun:test";
+import { WebSocket } from "ws";
+
+const hostname = process.env.HOST || "127.0.0.1";
+const port = parseInt(process.env.PORT || "0");
```

Comment:
we should always fix the port to 0 and get the dynamic port from `server`

Reply:
It seems the other WebSocket tests also may take in the `PORT` variable (referring to websocket-client.test.ts). Let me know if it's preferred that `PORT` does not get used though.